src/content/reference/react-dom/client/createRoot.md

```diff
@@ -369,7 +369,7 @@ root.render(<App />);
 The <CodeStep step={1}>onUncaughtError</CodeStep> option is a function called with two arguments:
 
 1. The <CodeStep step={2}>error</CodeStep> that was thrown.
-2. An <CodeStep step={3}>errorInfo</CodeStep> object that contains the <CodeStep step={4}>componentStack</CodeStep> of the error.
+2. An <CodeStep step={3}>errorInfo</CodeStep> object that contains the parent Component stack in <CodeStep step={4}>componentStack</CodeStep> of the error.
```

Comment:
The capitalization of only "Component" seems weird, seems like it should either be Component Stack or component stack.